cmds/umount/umount.go

```diff
@@ -0,0 +1,9 @@
+// Copyright 2017 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
```

Comment:
Add a man-page here similar to `mount` so that it automatically shows up in [godocs](https://godoc.org/github.com/u-root/u-root/cmds/mount).